training/src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,9 +1,15 @@
 package wolox.training.controllers;
 
+<<<<<<< HEAD
```

Comment:
You commited this file without solve the conflict (There are some `<<<<< HEAD` and `>>>>>> master`